src/lib/salad/mhash.h

```diff
@@ -75,6 +79,26 @@
 
 #define mh_unlikely(x)  __builtin_expect((x),0)
 
+#define mh_mmi(p) (UINT64_C(0xFFFFFFFFFFFFFFFF) / (p) + 1)
```

Comment:
What mmi stands for?

Reply:
https://en.wikipedia.org/wiki/Modular_multiplicative_inverse